bndtools.api/src/org/bndtools/api/central/ICentral.java

```diff
@@ -0,0 +1,22 @@
+package org.bndtools.api.central;
+
+import org.bndtools.api.ModelListener;
+import org.eclipse.jdt.core.IJavaProject;
+import org.osgi.annotation.versioning.ProviderType;
+
+import aQute.bnd.build.Project;
+
+@ProviderType
+public interface ICentral {
```

Comment:
```suggestion
public interface IWorkspaceManger {
```

Reply:
Since the name is something I inherited, and this is a rather central piece of code (pardon the pun) I don't feel right approving a change by myself.

But if we're going to change the name, I think I would prefer to change it to something like `BndWorkspaceManager`. There has been one too many times while working on Bndtools that I have gotten confused between an Eclipse workspace object and a Bnd workspace object, and that consumes precious brain cycles. And that's from a seasoned contributor's perspective - I pity the newbie coming along wanting to make a contribution to the Bndtools codebase for the first time and trying to comprehend the difference between a Workspace and a Workspace... (I was that newbie once, and I remember it being a painful experience...)

@bjhargrave expressed a strong dislike for the `I`-prefix naming convention for interfaces when he reviewed this PR (https://github.com/bndtools/bnd/pull/5036#discussion_r787112643). I personally do not feel quite so strongly about it either way - as I mentioned, although Eclipse/IDE can easily tell you if it's an interface, I see some utility in that you can see it's an interface without having to hover over it or do any kind of code navigation. Perhaps I'll let @pkriens have the last word here on whether we go for `IBndWorkspaceManager` or `BndWorkspaceManager`.

Reply:
I don't mind the I prefix, personally don't use it but if the projects wants it... as you said its redundant and I prefer to follow Java convention that the Interface is the name WHAT it is and the class should mention HOW it is implemented e.g. with `List` (Interface) and `ArrayList` (an implementation using an array as its backing store)...

So for `BndWorkspaceManger` (the thing) `CentralBndWorkspaceManger` (aka `Central`) and implementation that use **one** Workspace ever.

Reply:
Maybe call it BndManager? Not only is it even shorter, it will probably be mainly used by non-workspace users?

And as always, <=>